sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -75,6 +95,12 @@ export interface ChangeFeedOptions {
     startTime?: Date;
 }
 
+// @public
+export abstract class ChangeFeedPullModelIterator<T> {
+    abstract get hasMoreResults(): boolean;
```

Comment:
This has different casing as ReadNextAsync, is this by design?

Reply:
I will confirm the final public API names with the team.